decidim-participatory_processes/app/models/decidim/participatory_process.rb

```diff
@@ -69,5 +78,13 @@ def hashtag
     def to_param
       slug
     end
+
+    def self.private_processes
+      where(private_space: true)
+    end
+
+    def self.non_private_processes
```

Comment:
Same as in assemblies: We should probably be extending this instead: https://github.com/decidim/decidim/blob/master/decidim-core/lib/decidim/participable.rb#L82